es-bs-base/scss/_variables.scss

```diff
@@ -274,7 +276,7 @@ $embed-responsive-aspect-ratios: join(
 // Font, line-height, and color for body text, headings, and more.
 
 // stylelint-disable value-keyword-case
-$font-family-sans-serif:      "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
+$font-family-sans-serif:      "Inter", "Helvetica Neue", Roboto, "Segoe UI", -apple-system, BlinkMacSystemFont, Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
```

Comment:
In the absense of "Inter" wouldn't it be better to fall back to "San Francisco" vs. "Helvetica Neue"? What's the rational for moving `-apple-system` lower in the fallback order?

Reply:
I have a design friend who is obsessed with typography. This is the order he recommended, I'm paraphrase but he said "dont put system fonts before web fonts" "Helvetica is a paid version of Roboto, if I paid for a font I want to see it."

That said we can play with the order more I don't feel strongly about this change.